es-home/vite.config.js

```diff
@@ -1,8 +1,33 @@
 import reactRefresh from '@vitejs/plugin-react';
 import { defineConfig } from 'vite';
-
+import fs from "fs";
+import path from "path";
 const urlPath = process.env.URLPATH || ''
 
+function listTsFiles(options) {
+    const { srcDir, ignoredPaths = [], ignoreSubDirs = false } = options;
+    let entries = {};
+
+    function exploreDirectory(dir) {
```

Comment:
'listTsFiles' constructs a dictionary that maps each fileName within /es-runtime/src to its corresponding fullPath, effectively designating them as entrypoints for Vite's bundler. This mapping ensures that Vite accurately recognizes and incorporates all files during the bundling process, preventing it from inadvertently omitting or obscuring any files.